src/brsTypes/Float.ts

```diff
@@ -26,7 +27,8 @@ export class Float implements Numeric, Comparable, Boxable {
      * @param value the value to store in the BrightScript float, rounded to 32-bit floating point
      *              precision and maintaining only seven significant digits of accuracy.
      */
-    constructor(value: number) {
+    constructor(value: number | Long) {
+        if (value instanceof Long) value = value.toNumber();
```

Comment:
oh wow I'm surprised `prettier` allowed this 😉

Reply:
I'd never do that on purpose. Will fix.